script/pgsqlms

```diff
@@ -1173,15 +1217,21 @@ sub pgsql_validate_all {
 
     chomp $PGVERSION;
 
-    $PGVERSION =~ /^(\d+)\.(\d+)$/;
-    $PGVERNUM = $1 * 10000 + $2 * 100;
+    $PGVERSION =~ /^(\d+)(?:\.(\d+))?$/;
+
+    $PGVERNUM = $1 * 10000;
+
+    # postgresql >= 10 do not include the 2nd num in the major release
+    $PGVERNUM += $2 * 100 if $1 < 10;
 
     if ( $PGVERNUM < 90300 ) {
         ocf_exit_reason( "PostgreSQL %s not supported. Require 9.3 and more",
             $PGVERSION );
         exit $OCF_ERR_INSTALLED;
     }
 
+    $PGXLOGDUMP = "$bindir/pg_waldump" if $PGVERNUM >= $PGVER_10;
+
     # check binaries
     unless ( -x $PGCTL and -x $PGPSQL and -x $PGCTRLDATA and -x $PGISREADY ) {
```

Comment:
It's not related to this PR in particular but is there a reason not to check for $PGXLOGDUMP here ?